site/content/index.md

```diff
@@ -53,18 +53,22 @@ plugin.  Every tool you use with Go can be used with Magefiles.
 ```
 mage [options] [target]
 Options:
-  -f    force recreation of compiled magefile
-  -h    show this help
+  -clean
```

Comment:
thanks for fixing this up... I should write a test to make sure it stays up to date.

Reply:
Probably best for another thread rather than directly related to this change, but it's likely possible to generate the documentation examples from the actual running program?